releases/release-1.12/release-1.12.md

```diff
@@ -0,0 +1,136 @@
+# Kubernetes 1.12 Release Schedule
+
+*Handy Links*
+
+* [Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.12/release_team.md)
+* Zoom: join Google Group to receive meeting invite
+* [Slack](https://kubernetes.slack.com/messages/sig-release/)
+* [Forum](https://groups.google.com/forum/#!forum/kubernetes-sig-release)
+* [Feature Tracking Sheet](http://bit.ly/k8s112-features) **TODO: needs created, and short url'd**
+* [Milestone Process](https://github.com/kubernetes/community/blob/master/contributors/devel/release/issues.md)
+* [Meeting Minutes](http://bit.ly/k8s112-minutes) **TODO: needs created, and short url'd**
+* [CI Signal Report](http://bit.ly/k8s112-cisignal) **TODO: needs created, and short url'd**
+* [Retrospective Document](http://bit.ly/k8s112-retro) **TODO: needs created, and short url'd**
+
+**tl;dr** The 1.12 release cycle begins on **Monday, July 9rd, 2018**, and ends on release day, **TBD, end of Sept or begin of Oct**.  Feature freeze is **TBD**.  Code freeze should begin **TBD** and ends **TBD**, but [may be vary depending on CI Signal](#conditionally-shortened-code-freeze). Docs must be completed and reviewed by **TBD**.
+
+## Notes About this Release
+
+* The Release Team will be using a conditional process to potentially shorten Code Freeze, [see below](#conditionally-shortening-code-freeze).
+* The feature process is remaining as it has in prior releases.
+* Features that don't have complete code and tests by [Code Freeze](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.12/release-1.12.md#code-freeze) may be disabled by the release team before cutting the first beta.
+* The release team will escalate [release-master-blocking](https://k8s-testgrid.appspot.com/sig-release-master-blocking) failures to SIGs throughout the cycle, not just near release cuts.
```

Comment:
and [release-master-upgrade](https://k8s-testgrid.appspot.com/sig-release-master-upgrade#gce-1.10-master-upgrade-master&show-stale-tests=)

Reply:
added